bare/MagicBareRnExample/index.js

```diff
@@ -1,8 +1,8 @@
 /**
  * @format
  */
-
 import {AppRegistry} from 'react-native';
+import "./shim";
```

Comment:
Could we rename the shim to `./bitcoinShim`? cuz I anticipate we will have more shims files like this for each blockchain package exclusively.

Reply:
So slight problem here, `rn-nodeify` and its requisite [post install](https://github.com/magiclabs/react-native-demo/pull/32/commits/481de28fa88e19d6b8a46ac48cfe5356c93a0f83#diff-f005dea26626c10b4f7f2bc2388e43572f4cdefa83f972319b5d59f365e70be5R11) command won't let me change the name :-/. 

If I do, a new `shim.js` file is created upon the next `yarn install`.

Reply:
If you're curious as to how it all works, [here's a gist](https://gist.github.com/coreyphillips/4d45160fed016417a5f583f179c2cbdb) I found to be helpful.